README.md

```diff
@@ -2,30 +2,43 @@
 
 [![GitHub Release](https://img.shields.io/github/release/nhsuk/gp-finder.svg)](https://github.com/nhsuk/gp-finder/releases/latest/)
 [![Greenkeeper badge](https://badges.greenkeeper.io/nhsuk/gp-finder.svg)](https://greenkeeper.io/)
-
 [![Build Status](https://travis-ci.org/nhsuk/gp-finder.svg?branch=master)](https://travis-ci.org/nhsuk/gp-finder)
 [![Coverage Status](https://coveralls.io/repos/github/nhsuk/gp-finder/badge.svg?branch=master)](https://coveralls.io/github/nhsuk/gp-finder?branch=master)
 [![Known Vulnerabilities](https://snyk.io/test/github/nhsuk/gp-finder/badge.svg)](https://snyk.io/test/github/nhsuk/gp-finder)
 
-A GP surgery finder application to allow the user to easily navigate to a GP
+> A GP surgery finder application to allow the user to easily navigate to a GP
 surgery's online booking system.
 
+## Test environments
+
+As the application is being developed, every Pull Request has its own test
+environment automatically built and deployed to. The path to the environment
+is added as a comment to the PR in GitHub.
```

Comment:
This is not strictly true - the link is in the checks (as the Details for rancher/deployment check)

Reply:
Yeah but that hasn't changed. Add as an issue?

Reply:
My bad. Sorted this and alphabetised the docker config to make it easier to search through.